pyanaconda/users.py

```diff
@@ -166,6 +167,21 @@ def validatePassword(pw, user="root", settings=None, minlen=None):
 
     return (valid, strength, message)
 
+def check_name(in_name):
+    ign_cond = {r"^[\d]": "User name cannot start with a digit: ",
+                r"[^A-Za-z0-9\s]": "User name cannot contain any non-alphanumerical character: "}
```

Comment:
I think ``_`` is missing and the ``\s`` is extra here. Or can user name contain a space?